scrapi/harvesters/usgs.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import unicode_literals
+
+import json
+import logging
+from datetime import date, timedelta
+
+import six
+from nameparser import HumanName
+from scrapi import requests, settings
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import default_name_parser, compose, build_properties, datetime_formatter
+
+logger = logging.getLogger(__name__)
+
+
+def process_contributors(authors):
```

Comment:
I recommend you use the `default_name_parser` helper here: 
```python
names = [author['text'] for author in authors]
return default_name_parser(names)
```

You could then even put it inline in the schema, using the `compose` helper:
```
schema = {
    ...,
    'contributors': ('/contributors/authors', compose(
        default_name_parser, 
        lambda authors: [author['text'] for author in authors]
    )),
    ...
}
```